source/parameters/matter.md

```diff
@@ -0,0 +1,10 @@
+(s:matter.params)=
+## MatterP:  Definition of materials
```

Comment:
I am confused, `MatterP` is not referenced anywhere.

How does it relate to `MaterialP`/`ElementP`?

Reply:
Ah, there is a typo below...